temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowOutboundCallsInterceptorBase.java

```diff
@@ -156,6 +156,11 @@ public void upsertTypedSearchAttributes(SearchAttributeUpdate<?>... searchAttrib
     next.upsertTypedSearchAttributes(searchAttributeUpdates);
   }
 
+  @Override
+  public void upsertMemo(Map<String, Object> memo) {
+    next.upsertSearchAttributes(memo);
```

Comment:
```suggestion
    next.upsertMemo(memo);
```

Reply:
Tests passed so looks like we must be missing some test coverage?

Reply:
Yep added some test coverage for this, will need to think long term if there is some way I can test all the interceptors programatically